andela/denethor-ah-backend
# #163478810 Implement read time algorithm

#### What does this PR do?
- Implement the algorithm to calculate read time for an article

#### Description of Task to be completed?
- write tests
- write algorithm to calculate the read time

#### How should this be manually tested? 
- Clone the repo and pull this branch `feature/163478810/read-time-algorithm`
- check out the method `getReadTime` in `/api/controllers/articles`

#### What are the relevant pivotal tracker stories?
(https://www.pivotaltracker.com/story/show/163478810)
